.github/workflows/profiling.yaml

```diff
@@ -0,0 +1,120 @@
+---
+name: Performance Profiling
+on:
+  # Trigger on Issue created; Run against master branch
+  issues:
+    types: [opened, edited]
+    branches: 
+      - 'master'
+  
+  pull_request:
+    types: [labeled, edited]
+    branches:
+      - 'master'
+
+  # Trigger when an Issue is created or modified; Run against master branch only
+  issue_comment:
```

Comment:
Can you explain how to trigger it with a comment?

Reply:
I think currently _any_ comment will trigger it if the issue/PR is labelled `performance` (line 28).
I found a [discussion in github community where someone suggests `if (contains(...))` to match on the comment content](https://github.com/orgs/community/discussions/25389#discussioncomment-3247738), which we could maybe add here.

Reply:
We should definitely not trigger the job on _any_ comment. Let's implement ^ in a subsequent PR.